platform/build-faiss/build-faiss.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+git clone https://github.com/facebookresearch/faiss
+cd faiss
+
+# Configure paths and set environment variables
```

Comment:
nit: pointless comment. The problem with these types of comments is, over time, they introduce code smells.